### ISSUE TITLE
Deprecated selector in `foldingtext-for-atom\styles\foldingtext.less`

### ISSUE THREAD

**Issue by AdrianDroid** [FoldingText/foldingtext-for-atom#86]
In `foldingtext-for-atom\styles\foldingtext.less`: 

Starting from Atom v1.13.0, the contents of `atom-text-editor` elements are no longer encapsulated within a shadow DOM boundary. This means you should stop using `:host` and `::shadow` pseudo-selectors, and prepend all your syntax selectors with `syntax--`. To prevent breakage with existing style sheets, Atom will automatically upgrade the following selectors:

* `ft-token-input.has-token-selected > ft-text-input > atom-text-editor[mini]::shadow .cursors` => `ft-token-input.has-token-selected > ft-text-input > atom-text-editor[mini].editor .cursors`

Automatic translation of selectors will be removed in a few release cycles to minimize startup time. Please, make sure to upgrade the above selectors as soon as possible.

**Comment by jurowski** [comment]
Same issue here.

HogBay, I run my life on this plugin. did I hear right that you're not updating FoldingText for Atom further? I'm HUGE about it and would love to help improve it.

-SJ

**Comment by jessegrosjean** [comment]
Yes, I don't expect to make any more updates myself in foreseeable future, but would be happy to accept patches.

**Comment by aadrian** [comment]
Any news on this?

**Comment by iterating** [comment]
I'm also waiting to hear any news about this. The OPML format works very well with using a mindmapper to organize and rearrange ideas and importing it into FoldingText to read or publish.
